lib/seahorn/BvOpSem2MemRepr.cc

```diff
@@ -347,8 +347,20 @@ OpSemMemLambdaRepr::MemCpy(PtrTy dPtr, PtrTy sPtr, unsigned len,
     MemValTy srcMem = memTrsfrRead;
 
     if (len > 0) {
-      unsigned bytesToCpy = len - wordSzInBytes;
-      PtrTy dstLast = m_memManager.ptrAdd(dPtr, bytesToCpy);
+      unsigned lastByteToCopy;
+      unsigned remainderBytes;
+      if (m_memManager.isIgnoreAlignment()) {
+        // if alignment is ignored, we treat it as alignment of 1
+        lastByteToCopy = len - 1;
+        remainderBytes = 0;
+      } else {
+        unsigned wordsToCopy = (len / wordSzInBytes);
+        // -- -1 because ptrInRangeCheck is inclusive
+        lastByteToCopy = (wordsToCopy - 1) * wordSzInBytes;
```

Comment:
This is really `lastAlignedBytePosToCopy` isn't it? Do you think that is a better name?

Reply:
done. will commit soon.